python/lsst/pipe/tasks/coaddInputRecorder.py

```diff
@@ -112,7 +112,7 @@ def addCalExp(self, calExp, ccdId, nGoodPix):
             self._setExposureInfoInRecord(exposure=calExp, record=record)
             if self.task.config.saveCcdWeights:
                 record.setD(self.task.ccdWeightKey, 1.0)  # No weighting or overlap when warping
-            record.set(self.task.ccdFilterKey, calExp.getFilter().getName())
+            record.set(self.task.ccdFilterKey, calExp.getFilterLabel().bandLabel)
```

Comment:
This one's a bit problematic, since IIRC the `obs_subaru` coadd code depends on this key distinguishing HSC-R from HSC-R2 (previously accomplished through the use of `afw_name`). So I think you may actually need the physical filter here.

Reply:
This is now correctly `physical_filter`, but that opened up a whole host of other questions that @yalsayyad helped me sort out.